src/styles/BookList.css

```diff
@@ -0,0 +1,17 @@
+.bookContainer {
+  margin: auto auto;
+  background-color: #fafafa;
+}
+
+.list {
+  margin: auto auto;
+  border-bottom: solid 1px #e8e8e8;
+  width: 90%;
+  padding-inline-start: 0;
+  padding-top: 1px;
+  padding-bottom: 1%;
+}
+
+.list li {
+  list-style-type: none;
+}
```

Comment:
Great design @tomasesquivelgc 🤝

- [ ] As stated in the project requirements, we are supposed to replicate the given design as much as possible. Kindly ensure the highlighted section replicates the given design as much as possible. You may separate these elements properly.
- Your design
![your-design](https://github.com/tomasesquivelgc/React-Redux-bookstore/assets/63917725/1571ec83-1f2d-4523-b0ba-c72001f449e6)
- Given design
![Given design](https://github.com/tomasesquivelgc/React-Redux-bookstore/assets/63917725/edac34fd-e164-45f9-8cdf-51a010555fc1)